src/Groups/homomorphisms.jl

```diff
@@ -570,7 +570,7 @@ function isomorphism(::Type{FPGroup}, G::GAPGroup; on_gens::Bool=false)
    # Known isomorphisms are cached in the attribute `:isomorphisms`.
    isos = get_attribute!(Dict{Tuple{Type, Bool}, Any}, G, :isomorphisms)::Dict{Tuple{Type, Bool}, Any}
    return get!(isos, (FPGroup, on_gens)) do
-     if is_trivial(G)
+     if is_trivial(G) && ngens(G) == 0
 # TODO: remove this special treatment as soon as the change from
 #       https://github.com/gap-system/gap/pull/5700 is available in Oscar
 #       (not yet in GAP 4.13.0)
```

Comment:
https://github.com/gap-system/gap/pull/5700 was backported to gap 4.13.1 which is available in Oscar right now, so you could remove this special case altogether if I am not mistaken here

Reply:
Indeed, that seems to work. I just pushed a corresponding commit. It works locally, let's see if it also passes CI.